### ISSUE TITLE
[all] The rules document links on the npm home page report 404 error.

### ISSUE THREAD

**Issue by FeelyChau** [typescript-eslint/typescript-eslint#2310]
**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/<rule>": ["<setting>"]
  }
}
```

```TS
// your repro code case
```

**Expected Result**
The rules document should jump to the correct page.
**Actual Result**
404 error.
**Additional Info**
The relative path of the document on the npm home page causes a jump error.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.6.1` |

**Comment by bradzacher** [comment]
yeah if anyone can figure out how to make it work I'd appreciate it.

We've got the directory set in the package.json config, but for some reason NPM doesn't respect it.

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/package.json#L24

**Comment by FeelyChau** [comment]
I checked the document about the README.md here: https://docs.npmjs.com/about-package-readme-files
As the document said, `An npm package README file must be in the root-level directory of the package.`. 
Cloud we change the README file location to adapt npm?

**Comment by FeelyChau** [comment]
Sorry, I made a mistake. The README file is in the package root directory correctly.

**Comment by bradzacher** [comment]
Quick google looks like this is just a KP with NPM.
Example question with no solution: https://npm.community/t/relative-link-in-readme-of-monorepo/3806

🤷‍♂️
Maybe one day they'll fix it to support their documented `repository.directory` config option.
https://docs.npmjs.com/files/package.json#repository

For now it looks like there's nothing we can to - people just have to read the docs in the github repo itself.
